ganttproject/src/net/sourceforge/ganttproject/export/ExporterToCSV.java

```diff
@@ -18,27 +18,74 @@
  */
 package net.sourceforge.ganttproject.export;
 
-import java.awt.Component;
+import biz.ganttproject.core.option.EnumerationOption;
+import biz.ganttproject.core.option.GPAbstractOption;
+import biz.ganttproject.core.option.GPOption;
+import biz.ganttproject.core.option.GPOptionGroup;
+import biz.ganttproject.impex.csv.GanttCSVExport;
+import net.sourceforge.ganttproject.GPLogger;
+import net.sourceforge.ganttproject.GanttProject;
+import net.sourceforge.ganttproject.io.CSVOptions;
+import net.sourceforge.ganttproject.language.GanttLanguage;
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.Status;
+
+import java.awt.*;
 import java.io.BufferedOutputStream;
 import java.io.File;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.OutputStream;
+import java.util.Arrays;
 import java.util.List;
 
-import net.sourceforge.ganttproject.GPLogger;
-import net.sourceforge.ganttproject.GanttProject;
-import net.sourceforge.ganttproject.language.GanttLanguage;
+public class ExporterToCSV extends ExporterBase {
+  static class FileTypeOption extends GPAbstractOption<String> implements EnumerationOption {
+    static final String[] FILE_FORMAT_ID = new String[]{"impex.csv.fileformat.csv", "impex.xls.fileformat.xls"};
```

Comment:
reminder: 
1. rename `impex.xks.fileformat.xls` to `impex.csv.fileformat.xls`
It is handy when related keys go next to each other in the sorted order in i18n.properties
1. Plural: `IDS`